forem/forem
# Add paste image

## What type of PR is this? (check all applicable)


## Description
Add ability to paste images along side drag and drop

## Related Tickets & Documents
closes #10210

## QA Instructions, Screenshots, Recordings

![forem__paste-image-directly](https://user-images.githubusercontent.com/3534427/95455158-5a85d800-0965-11eb-88bb-859a18fd621b.gif)

1. Copy an image into your clipboard.
2. Paste it into article body
3. See it appear

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

![Mad Scientist](https://thumbs.gfycat.com/CluelessSeveralAoudad-size_restricted.gif)
